activities/domain/v1/domain.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.domain.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.domain.v1";
+option go_package = "activities.domain.v1";
+
+import "google/protobuf/timestamp.proto";
+import "global/domain/v1/domain.proto";
+import "global/selection/v1/reference.proto";
+
+
+message Activity {
+  string id = 1;
+
+  // activity type
+  ActivityType.Type activity_type = 2;
+
+  // who causes the activity
+  io.spoud.sdm.global.selection.v1.EntityRef triggered_by = 4;
```

Comment:
Sooo the cause by could me not only a user ? that what you mean here?

Reply:
yes. I though like e.g "Agent discover a new Data Port X" or something similar. it that make sense to you?

Reply:
No, because the agent is a technical user which is a subject. So I would have subject here.

Reply:
so all the change will be made by User, UserGroup, TechnicalUser?

Reply:
yes